File: tests/Pit.Tests/common/Assert.fs
namespace Pit

[<JsIgnore(IgnoreNamespace=true)>]
module Assert =

    [<JsIgnore(IgnoreTuple=true)>]
    let AreEqual (fnName:string) (a1:obj) (a2: obj) = ()

    [<JsIgnore(IgnoreTuple=true)>]
    let AreNotEqual (fnName:string) (a1:obj) (a2: obj) = ()

    [<JsIgnore(IgnoreTuple=true)>]
    let IsNull (fnName:string) (a1:obj) = ()

    [<JsIgnore(IgnoreTuple=true)>]
    let IsNotNull (fnName:string) (a1:obj) = ()

File: tests/Pit.Tests/core/web/assert.js
registerNamespace('Pit');
Pit.Assert = (function () {
    function Assert() {
    }

    Assert.prototype.AreEqual = function (fnName, a1, a2) {
        var res = a1 === a2;
        if (res) {
            console.log("Function - " + fnName + " Result - " + res);
        }
        else {
            console.error("Function - " + fnName + " Failed");
        }
    };

    Assert.prototype.AreNotEqual = function (fnName, a1, a2) {
        var res = a1 !== a2;
        if (!res) {
            console.log("Function - " + fnName + " Result - " + res);
        }
        else {
            console.error("Function - " + fnName + " Failed");
        }
    };

    Assert.prototype.IsNull = function (fnName, a1) {
        var res = a1 === null;
        if (res) {
            console.log("Function - " + fnName + " Result - " + res);
        }
        else {
            console.error("Function - " + fnName + " Failed");
        }
    };

    Assert.prototype.IsNotNull = function (fnName, a1) {
        var res = a1 !== null;
        if (res) {
            console.log("Function - " + fnName + " Result - " + res);
        }
        else {
            console.error("Function - " + fnName + " Failed");
        }
    };

    return Assert;
})();
var Assert = new Pit.Assert();